qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1325,3 +1326,78 @@ def check_restrictions(self, restrictions):
                 for col in restriction.columns}
 
         return cols.difference(self.categories())
+
+    def _get_accession_numbers(self, column):
+        """Return the accession numbers stored under the given column
```

Comment:
under the given column -> in \`column\` 

As it stands it isn't incorrect, but it reads kinda funny.